pkg/cmd/test/test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 PingCAP, Inc.
```

Comment:
Seem like we do not need the test cmd any more. Cc @amyangfei

Reply:
Yes, it can be removed

Reply:
Removed! Thanks for your review!